src/components/Layouts/WomenInSpace/styles.scss

```diff
@@ -0,0 +1,15 @@
+body {
+  line-height: 1.6;
+  margin: 20px;
+  background-color: #333;
+  color: whitesmoke;
+}
+
+h1 {
```

Comment:
can we use a class name and not h1 please? I found out this morning that this will interact with other h1s on the page